cherryontech/ywgc
# Ywgc 4 setup storybook

## Work done
- Installed Storybook
- Removed Storybook defaults
- Added an action to Primary button

## Manual testing instructions
Run "npm run storybook" in terminal and it should open in the browser 
- defaults and how-to for Storybook should no longer appear
- the only story should be for the Primary button
- 4 controls should be visible for the button
- when the button is clicked, the 'clicked' action should appear in the actions tab
